lib/matplotlib/patches.py

```diff
@@ -1451,7 +1451,10 @@ def _make_verts(self):
 
 
 docstring.interpd.update(
-    FancyArrow="\n".join(inspect.getdoc(FancyArrow.__init__).splitlines()[2:]))
+    FancyArrow="\n".join(
+        inspect.getdoc(FancyArrow.__init__).splitlines()[2:]
```

Comment:
I guess that's `(inspect.getdoc(...) or "")` if you want to be shorter (and likewise below), but not insisting on it.
From a quick test it seems that cpython also sets docstrings to None on -OO, so that may not be a PyPy-only issue, in fact?  (In which case the comment in scale.py can be changed.)

Reply:
It seems to specifically be `__init__` that CPython `getdoc` still provides a docstring for, or possibly whatever inherits from a C-level function.

Reply:
That does work as well.